arcdps_mock/main.cpp

```diff
@@ -43,9 +47,111 @@ struct ArcModifiers
+	ImVec4(0.380000f,1.000000f,1.000000f,1.000000f),
+	ImVec4(0.500000f,0.470000f,0.480000f,1.000000f),
+	ImVec4(0.250000f,0.220000f,0.230000f,1.000000f),
+	ImVec4(0.000000f,0.000000f,5.688562f,-0.000000f)
+};
+ImVec4 profColsBase[9] { // size of enum `Prof`
+	ImVec4(0.340000f,0.300000f,0.360000f,0.490000f),
+	ImVec4(0.040000f,0.870000f,1.000000f,0.430000f),
+	ImVec4(1.000000f,0.830000f,0.240000f,0.430000f),
+	ImVec4(0.890000f,0.450000f,0.160000f,0.430000f),
+	ImVec4(0.530000f,0.870000f,0.040000f,0.430000f),
+	ImVec4(0.890000f,0.370000f,0.450000f,0.450000f),
+	ImVec4(0.970000f,0.220000f,0.220000f,0.430000f),
+	ImVec4(0.800000f,0.230000f,0.820000f,0.430000f),
+	ImVec4(0.020000f,0.890000f,0.490000f,0.430000f)
+};
+ImVec4 profColsHighlight[9] { // size of enum `Prof`
+	ImVec4(0.340000f,0.300000f,0.360000f,0.250000f),
+	ImVec4(0.040000f,0.870000f,1.000000f,0.210000f),
+	ImVec4(1.000000f,0.830000f,0.240000f,0.210000f),
+	ImVec4(0.890000f,0.450000f,0.160000f,0.210000f),
+	ImVec4(0.530000f,0.870000f,0.040000f,0.210000f),
+	ImVec4(0.890000f,0.370000f,0.450000f,0.280000f),
+	ImVec4(0.970000f,0.220000f,0.220000f,0.210000f),
+	ImVec4(0.800000f,0.230000f,0.820000f,0.210000f),
+	ImVec4(0.020000f,0.890000f,0.490000f,0.210000f)
+};
+ImVec4 subgroupColsBase[15] { // max amount of subgroups (currently 15), defined by gw2
+	ImVec4(0.340000f,0.300000f,0.360000f,0.490000f),
+	ImVec4(0.970000f,0.140000f,0.140000f,0.430000f),
+	ImVec4(0.140000f,0.450000f,0.970000f,0.430000f),
+	ImVec4(0.640000f,0.140000f,0.970000f,0.430000f),
+	ImVec4(0.140000f,0.970000f,0.970000f,0.430000f),
+	ImVec4(0.970000f,0.970000f,0.140000f,0.430000f),
+	ImVec4(0.970000f,0.470000f,0.140000f,0.430000f),
+	ImVec4(0.140000f,0.970000f,0.140000f,0.430000f),
+	ImVec4(0.970000f,0.140000f,0.970000f,0.430000f),
+	ImVec4(0.470000f,0.400000f,0.190000f,0.430000f),
+	ImVec4(1.000000f,0.140000f,0.640000f,0.430000f),
+	ImVec4(0.800000f,1.000000f,0.000000f,0.430000f),
+	ImVec4(1.000000f,0.800000f,0.000000f,0.430000f),
+	ImVec4(0.000000f,0.800000f,1.000000f,0.430000f),
+	ImVec4(1.000000f,0.470000f,0.800000f,0.430000f)
+};
+ImVec4 subgroupColsHighlight[15] { // max amount of subgroups (currently 15), defined by gw2
+	ImVec4(0.340000f,0.300000f,0.360000f,0.250000f),
+	ImVec4(0.970000f,0.140000f,0.140000f,0.210000f),
+	ImVec4(0.140000f,0.450000f,0.970000f,0.210000f),
+	ImVec4(0.640000f,0.140000f,0.970000f,0.210000f),
+	ImVec4(0.140000f,0.970000f,0.970000f,0.210000f),
+	ImVec4(0.970000f,0.970000f,0.140000f,0.210000f),
+	ImVec4(0.970000f,0.470000f,0.140000f,0.210000f),
+	ImVec4(0.140000f,0.970000f,0.140000f,0.210000f),
+	ImVec4(0.970000f,0.140000f,0.970000f,0.210000f),
+	ImVec4(0.470000f,0.400000f,0.190000f,0.210000f),
+	ImVec4(1.000000f,0.140000f,0.640000f,0.210000f),
+	ImVec4(0.800000f,1.000000f,0.000000f,0.210000f),
+	ImVec4(1.000000f,0.800000f,0.000000f,0.210000f),
+	ImVec4(0.000000f,0.800000f,1.000000f,0.210000f),
+	ImVec4(1.000000f,0.470000f,0.800000f,0.210000f)
+};
```

Comment:
If you want this definitions in a different place, just say so (it not yet had a file just for global objects like this=

Reply:
nah this is fine. The arrays can all be marked const and static though?

Reply:
sure